src/effects_box.cpp

```diff
@@ -310,7 +310,7 @@ void setup(EffectsBox* self, app::Application* application, PipelineType pipelin
 
   adw_view_stack_page_set_icon_name(self->plugins_box_page, "emblem-music-symbolic");
 
-  // setting up the boxes we added t othe stack
+  // setting up the boxes we added t other stack
```

Comment:
Not sure what this sentence was supposed to mean, but this typo fix seems correct in itself.

Reply:
It should have been "to the stack" :smile: